### PR TITLE
Accessibility Improvements

### PR DESCRIPTION
**Pull request by t3chguy**
+ Fixes regression caused by https://github.com/matrix-org/matrix-react-sdk/pull/3562 to the scrollbars around the room list
+ Helps ARIA understand DateSeparator as a single separator instead of two with some text between them
+ Improve Accessibility of EmojiPicker and fix a PropTypes mismatch

### CODE REVIEW

File: src/components/views/emojipicker/ReactionPicker.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class ReactionPicker extends React.Component {
         mxEvent: PropTypes.object.isRequired,
         onFinished: PropTypes.func.isRequired,
         closeMenu: PropTypes.func.isRequired,
-        reactions: PropTypes.object.isRequired,
+        reactions: PropTypes.object,
```
Comment (turt2live) [inline src/components/views/emojipicker/ReactionPicker.js]: hmm?

File: src/components/views/emojipicker/Search.js [hunk 2]
```diff
@@ -35,13 +35,22 @@ class Search extends React.PureComponent {
     }
 
     render() {
+        let rightButton;
+        if (this.props.query) {
+            rightButton = (
+                <button onClick={() => this.props.onChange("")}
+                        className="mx_EmojiPicker_search_icon mx_EmojiPicker_search_clear"
```
Comment (turt2live) [inline src/components/views/emojipicker/Search.js]: 4 character indent, not aligned (for reasons I still don't understand)

File: src/components/views/emojipicker/ReactionPicker.js (hunk 1 above)
Comment (t3chguy) [reply src/components/views/emojipicker/ReactionPicker.js]: this comes from MessageActionBar which has PropTypes.object, ReactionPicker has null-guards yet the PropType says its Required. MessageActionBar defaults to undefined so causes proptypes error

Comment (turt2live) [reply src/components/views/emojipicker/ReactionPicker.js]: ah, alright then

**Pull request merged**